plutus-core/executables/Common.hs

```diff
@@ -559,3 +568,66 @@ runPrintExample getFn (ExampleOptions (ExampleSingle name)) = do
     T.putStrLn $ case lookup name examples of
         Nothing -> "Unknown name: " <> name
         Just ex -> PP.render $ prettyExample ex
+
+
+---------------- Print the cost model parameters ----------------
+
+runDumpModel :: IO ()
+runDumpModel = do
+    let params = fromJust PLC.defaultCostModelParams
+    BSL.putStr $ Aeson.encode params
+
+
+---------------- Print the type signatures of the default builtins ----------------
+
+-- Some types to represent signatures of built-in functions
+type PlcTerm = PLC.Term PLC.TyName PLC.Name PLC.DefaultUni PLC.DefaultFun ()
+type PlcType = PLC.Type PLC.TyName PLC.DefaultUni ()
+data QVarOrType = QVar String | Type PlcType  -- Quantified type variable or actual type
+
+data Signature = Signature [QVarOrType] PlcType  -- Argument types, return type
+instance Show Signature where
```

Comment:
Can we at least use `Pretty`? I suspect these things would benefit from slightly better layout.